dpnp/backend/extensions/lapack/gesv.cpp

```diff
@@ -83,12 +91,69 @@ static sycl::event gesv_impl(sycl::queue &exec_q,
         throw;
     }
```

Comment:
I guess allocation memory for `ipiv` can be moved out of any ifdef block, because it's common.